themes/classic/_dev/css/modules/checkout.scss

```diff
@@ -0,0 +1,84 @@
+#checkout {
```

Comment:
what is the point of having #checkout #checkout-cart-summary? it is common mistake, too much code